packages/airnode-deployer/terraform/airnode/aws/templates/httpSignedDataApiGateway.yaml.tpl

```diff
@@ -35,10 +35,7 @@ components:
 
     EndpointRequestExample:
       summary: Endpoint request example
-      value: { "parameters": { "from": "EUR", "amount": 5 } }
-    EndpointResponseExample:
-      summary: Endpoint response example
-      value: { "response": 20 }
+      value: { "encodedParameters": "0x31737300000000000000000000...0000000000000000000000" }
```

Comment:
It doesn't matter that this value has been truncated, right?

Reply:
No, it's just an example.